tests/test_cases/test_inertial_writes/inertial_writes_tests.py

```diff
@@ -51,8 +47,6 @@ async def test_writes_on_timer_seen_on_edge(dut):
     expect_fail = False
 elif SIM_NAME.startswith(("riviera", "modelsim")) and vhdl:
     expect_fail = False
-elif SIM_NAME.startswith("verilator"):
-    expect_fail = False
```

Comment:
Verilator now acts like most other simulators in this mode, where because writes are cached in Python until ReadWrite, then applied inertially, you can't read back what you just wrote.